package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "nestx-amqp",
-  "version": "1.3.1",
+  "version": "1.4.0",
```

no need to manual update version in `package.json`

`semantic-release` will perform commit message analyzing in ci

```suggestion
  "version": "1.3.1",
```